pkgs/build-support/node/fetch-yarn-deps/default.nix

```diff
@@ -150,4 +147,7 @@ in
         );
     in
     lib.setFunctionArgs f (lib.functionArgs f) // { inherit tests; };
+in
+{
+  inherit fixup-yarn-lock prefetch-yarn-deps fetchYarnDeps;
```

Comment:
I would probably not place an inherit here and just use the existing attr set.

And I am also very confused how prefetch-yarn-deps worked before.

Reply:
> I would probably not place an inherit here and just use the existing attr set.

Done.

> And I am also very confused how prefetch-yarn-deps worked before.

If you are confused about how the expression worked in terms of Nix, then yes it was confusing to me too at the beginning - the arguments to the expression included attributes that were reused by other attributes in the main attribute set returned by the expression.

Reply:
And now I reverted to the same scheme that was before.